res/values-it/strings.xml

```diff
@@ -16,37 +16,37 @@
 	<string name="confirm_delete_yes">sì</string>
 	<string name="confirm_delete_no">no</string>
 	<string name="options_title">Opzioni Titolo</string>
-	<string name="options_delete">cancellare</string>
-	<string name="confirm_play">giocare</string>
-	<string name="confirm_send">inviare</string>
-	<string name="sendMail_subject">Il tuo record dal telefono</string>
-	<string name="sendMail_body">Si tratta di un record di chiamata dal telefono</string>
-	<string name="about_title">circa</string>
-	<string name="about_content">Call Recorder da Kobi Krasnoff</string>
-	<string name="about_close_button">chiudere</string>
-	<string name="send_mail">Inviare e-mail</string>
-	<string name="withheld_number">Numero trattenuti</string>
-	<string name="menu_record_is_now_enabled">L\'applicazione è ora permettono di registrare le chiamate</string>
-	<string name="menu_record_is_now_disabled">L\'applicazione è ora non consentono di registrare le chiamate</string>
-	<string name="no_records_message">Non ci sono registrazioni in questo telefono.\nPer attivare la capacità record di aprire il menu principale e scegliete \"Abilita Record\".\nDopo che l\'applicazione registra tutte le chiamate in entrata e in uscita.</string>
-	<string name="radio_Enable_record">Abilita Record</string>
-	<string name="radio_Disable_record">Disattiva Record</string>
-	<string name="textview_startup">Si prega di scegliere se l\'applicazione registra le chiamate.\nÈ sempre possibile modificare la scelta premendo il tasto menu di Android.</string>
-	<string name="dialog_welcome_screen">Benvenuto schermo</string>
-	<string name="dialog_no_memory">Non vi è alcuna scheda di memoria nel dispositivo.\nSi prega di installare uno.</string>
-    <string name="dialog_close">chiudere</string>
+	<string name="options_delete">cancella</string>
+	<string name="confirm_play">riproduci</string>
+	<string name="confirm_send">invia</string>
+	<string name="sendMail_subject">La tua registrazione dal telefono</string>
+	<string name="sendMail_body">Una registrazione di una chiamata</string>
+	<string name="about_title">informazioni</string>
+	<string name="about_content">Registratore chiamate da Kobi Krasnoff</string>

```

Comment:
It appears you did not mention Raul Robledo (riul88).

See https://github.com/riul88/call-recorder-for-android/blob/master/res/values/strings.xml#L25